content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+
+If you are interested in weighing in on what kinds of educational content you
+would like to see from us, we'd be grateful if you'd fill out
+[**this survey**](https://docs.google.com/forms/d/e/1FAIpQLSdmwjs0ZkxDdODfZTvSwP2bVW4JAVVdxiYhQPyW5dSbsZC8qg/viewform?pli=1)
+to help us plan! 🙏🏼
+
+<external-link
+href="https://www.meetup.com/DVC-Community-Virtual-Meetups/events/279447414/"
+title="DVC Learn - Getting Started: Experiments"
+description="The next DVC Learn Meetup taught by Melecia McGregor designed to get you started with DVC Experiments"
+link="https://meetup.com"
+image="/uploads/images/2021-07-16/dvc_learn.png"/>
+
+## Data Science Journal Article on Reproducibility Practices in Research
+
+New research presented in the
+[Data Science Journal](https://datascience.codata.org/) aims to provide best
+practices for providing reproducibility in research datasets. This is necessary
+to pinpoint the version of the dataset that grounds any research. In this work
+the authors reviewed 39 use cases from 33 organizations to arrive at six
+principles for versioning datasets. These include Revision, Release,
+<span style="color:purple">G</span>ranularity,
+<span style="color:purple">M</span>anifestation, Provenance and Citation. See
+the full work below. 👇🏼
+
+<external-link
+href="https://datascience.codata.org/articles/10.5334/dsj-2021-012/"
+title="Versioning Data is About More Than Revisions:  A Conceptual Framework and Proposed Priniciples"
+description="Authors analyze 39 use cases in 33 organziations to arrive at proposed principles when versioning data."
+link="https://datascience.codata.org"
+image="/uploads/images/2021-07-16/dsj.png"/>
+
+## June Office Hours Meetup
+
+The June Office Hours Meetup was 🔥! Amazing discussion on experiments ignited
+by [Sami Jawhar](https://www.linkedin.com/in/sami-jawhar-a58b9849/) of
+[Kenel](https://www.kernel.com/) around experiment use cases and workflows.  
+You can find the repo for his presentation
+[here](https://github.com/sjawhar/dvc-cloud-runner) and watch all the great
+<span style="color:purple">D</span>VC discussion below.
+
+https://youtu.be/DxZdWq3Weng
+
+# DVC News
+
+<span style="color:purple">S</span>ummer and vaccinations mean travel! ☀️💉 And
+that travel has enabled some of our team members to get together! Pictured below
+are Dmitry Petrov, Alexander Guschin, Max Shmakov, Mikhail Rozhkov, Sergey
+Kryukov, Mikhail Sveshnikov, and Guro Bokum... But not necessarily in that
+order.
+
+The first person to guess the correct order of our teammates starting from the
+upper right of the picture going clockwise, and post in the corresponding
+[Twitter](https://twitter.com/DVCorg) Heartbeat post will win some DVC SWAG!
+Hint: If you've bee wondering why there are random puple letters in this blog
+post, they're a clue to this cipher. 🧐
+
+![](/uploads/images/2021-07-17/team.jpg) _Team Meetup in Moscow!_
+
+## Jobs requiring DVC!
```

Comment:
it looks like we have it twice (I mean this section)

Reply:
Oops!  Thanks!